src/App.tsx

```diff
@@ -1,27 +1,59 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const handleGetAllGoods = useCallback(async () => {
+    const allGoods = await getAll();
```

Comment:
you didn't  predict case if getAll() throw an error, you can catch and handle it, page should work if request isn't successful

Reply:
Thanks, I forgot about it.